src/components/Captions/imagecaptions.stories.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import ImageCaptions from './imagecaptions'
+
+storiesOf('ImageCaption', module)
+  .add('Design And Made With Love', () => (
+    <ImageCaptions title="Design And Made With Love" description="French and Italian laces and silks... " />
+  ));
+  storiesOf('ImageCaption', module)
+  .add('Tailored to You', () => (
+    <ImageCaptions title="Tailored to You" description="An in-person experience will give you..." />
+  ));
+  storiesOf('ImageCaption', module)
+  .add('Flagship Boutiques', () => (
+    <ImageCaptions title="Flagship Boutiques" description="Exclusively shop our gowns... " />
+  ));
```

Comment:
For these two stories, it looks like the description is next to the title, instead of below it.

Reply:
@erinkelley27 not sure what you mean here? like in the figma file?